src/cli/lib/next-config.ts

```diff
@@ -0,0 +1,85 @@
+import { builders, loadFile, generateCode, writeFile } from 'magicast';
+
+import fs from 'node:fs/promises';
+import path from 'node:path';
+
+import { PACKAGE_NAME } from '../../constants.js';
+
+const COMMON_TEMPLATE = `
+
+// Everything below here added by the ${PACKAGE_NAME} CLI wizard.
+// You should probably clean this up.
```

Comment:
I think this is pretty creative. Does your average next.js user actually care, or should this be more like "feel free to clean this up"?